packages/connect-react/src/index.ts

```diff
@@ -1,7 +1,162 @@
-export function useAragon(): void {
-  return
+import React, {
+  createElement,
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react'
+import {
+  App,
+  ConnectOptions,
+  ConnectorDeclaration,
+  Organization,
+  Permission,
+  connect,
+} from '@aragon/connect'
+
+type ConnectProps = {
+  children: React.ReactNode
+  connector: ConnectorDeclaration
+  location: string
```

Comment:
`location` feels a bit vague, especially as a keyed React prop. It works a lot better as a positional argument's name and I don't have any better suggestions!

Reply:
What it be `organization`? I avoided it because we tend to use it for `Organization` instances, but I don’t think that’s a big issue.

In other places, I am trying to use `location` when both an Ethereum address and an ENS domain are accepted, and `address` when only an Ethereum address is accepted.

Reply:
I quite like `location` I think in the context of the Connect component should not be that vague. Otherwise how you feel about `identity`?

Reply:
`identity` feels a bit too far from the idea of “location of your organization” IMO, I would prefer to keep `location` until we find something better.

For now I think any of these would work:

- `address` (“address of the organization”, we could clarify in the docs that it accepts ENS domains)
- `organization` (we could specify in the docs that it accepts a string and not an `Organization`).
- `location`

Reply:
`location` sounds good—let's just make sure the concept of an organization's "location" (address or ENS domain resolving to a valid address) is in the docs!